containers/eap/src/test/java/org/jboss/qa/jcontainer/eap/test/EapConfigurationTest.java

```diff
@@ -0,0 +1,39 @@
+package org.jboss.qa.jcontainer.eap.test;
+
+import static org.junit.Assert.assertThat;
+
+import static org.hamcrest.CoreMatchers.hasItem;
+import static org.hamcrest.Matchers.equalTo;
+
+import org.jboss.qa.jcontainer.Configuration;
+import org.jboss.qa.jcontainer.eap.EapConfiguration;
+import org.jboss.qa.jcontainer.wildfly.WildflyConfiguration;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+import java.util.Arrays;
+import java.util.List;
+
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
+@RunWith(JUnit4.class)
+public class EapConfigurationTest {
+
+	@Test
+	public void javaOptsTest() {
```

Comment:
split this test method into multiple tests like:
- default
- set
- override
- xms
- xmx